crates/next-api/src/app.rs

```diff
@@ -1642,9 +1642,15 @@ impl Endpoint for AppEndpoint {
                 .clone_value();
 
             let client_relative_root = this.app_project.project().client_relative_path();
-            let client_paths = all_paths_in_root(output_assets, client_relative_root)
-                .await?
-                .clone_value();
+            let client_paths = async {
```

Comment:
Why do we need to wrap this in its own future rather than instrumenting the future directly?

Reply:
I did try that, but somehow couldn't make the types work. But `vc.into_future().instrument().await?` seems to work 👍